include/wx/private/webrequest_curl.h

```diff
@@ -87,6 +97,10 @@ class wxWebRequestCURL : public wxWebRequestImpl
     wxObjectDataPtr<wxWebResponseCURL> m_response;
     wxObjectDataPtr<wxWebAuthChallengeCURL> m_authChallenge;
     wxFileOffset m_bytesSent;
+    bool m_cancelPending;
+    bool m_hasInternalFail;
+    wxString m_internalFailMsg;
```

Comment:
AFAICS those 2 are redundant, i.e. `m_hasInternalFail == !m_internalFailMsg.empty()` must always be true. I'd remove `m_hasInternalFail` and just keep the other one if this is indeed correct, not for any optimization/space saving reasons, but just to avoid any chance of this invariant being somehow broken.

Reply:
As above these were both removed.